release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -311,9 +311,103 @@ The cluster network can be expanded to support the addition of nodes to the clus
 
 [id="ocp-4-13-storage-lvms-dual-stack-support"]
 ==== Dual-stack support for {lvms-first}
-
 In {product-title} {product-version}, {lvms} is supported in dual-stack for IPv4 and IPv6 network environments. For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network].
 
+[id="ocp-4-13-storage-user-managed-encryption"]
+==== User-managed encryption is generally available
+The user-managed encryption feature encrypts {product-title} node root volumes and the default storage class.
```

Comment:
```suggestion
The user-managed encryption feature encrypts {product-title} node root volumes and all the managed storage classes.
```
We also need update as we discussed in https://github.com/openshift/openshift-docs/pull/55014/files#r1152745609 .

Reply:
Done.